x/leverage/keeper/token.go

```diff
@@ -2,46 +2,13 @@ package keeper
 
 import (
 	"fmt"
-	"strings"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 
 	"github.com/umee-network/umee/v2/x/leverage/types"
 )
 
```

Comment:
The denom-manipulating functions (`FromUTokenToTokenDenom`, `FromTokenToUTokenDenom`) are moves to `types` package (`ToTokenDenom`, `ToUTokenDenom`).

The boolean-returning, registry checking functions (`IsAcceptedToken` and `IsAcceptedUToken`) have been replaced with error-returning functions in `validate.go`.